velox/functions/prestosql/window/tests/SimpleAggregatesTest.cpp

```diff
@@ -122,5 +122,27 @@ TEST_F(StringAggregatesTest, nonFixedWidthAggregate) {
   testWindowFunction(input, "max(c2)", kOverClauses);
 }
 
+class DecimalAggregatesTest : public WindowTestBase {
```

Comment:
It might be better to make this like StringAggregatesTest (which doesn't have any methods in it and a single TEST_F). Make the testAggregate a lambda in that fixture.

Reply:
Updated, thanks for the suggestion.